colcon_bundle/verb/assets/v1_setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+# You must set BUNDLE_CURRENT_PREFIX before sourcing this file
+export PATH="$BUNDLE_CURRENT_PREFIX/usr/sbin:$BUNDLE_CURRENT_PREFIX/usr/bin:$BUNDLE_CURRENT_PREFIX/usr/local/bin:$BUNDLE_CURRENT_PREFIX/sbin:$BUNDLE_CURRENT_PREFIX/bin:$PATH"
+export LD_LIBRARY_PATH="$BUNDLE_CURRENT_PREFIX/lib:$BUNDLE_CURRENT_PREFIX/usr/lib:$BUNDLE_CURRENT_PREFIX/lib/x86_64-linux-gnu:$BUNDLE_CURRENT_PREFIX/usr/lib/x86_64-linux-gnu:$BUNDLE_CURRENT_PREFIX/usr/lib/x86_64-linux-gnu/mesa:$LD_LIBRARY_PATH:$BUNDLE_CURRENT_PREFIX/lib/arm-linux-gnueabihf:$BUNDLE_CURRENT_PREFIX/usr/lib/arm-linux-gnueabihf"
+
+export LC_ALL=C.UTF-8
+
+if [ -f $BUNDLE_CURRENT_PREFIX/opt/ros/kinetic/setup.sh ]; then
```

Comment:
quotes

=> all shell files should be linted with shellcheck for this kind of issues